poetry.lock

```diff
@@ -556,7 +556,7 @@ python-versions = "*"
 name = "psycopg2"
 version = "2.8.6"
 description = "psycopg2 - Python-PostgreSQL Database Adapter"
-category = "main"
+category = "dev"
```

Comment:
Somehow we seem to be differently configured or something as the two of us seem to constantly change these back and forth between main and dev.
I haven't looked into it, but do you have any idea?

Reply:
I'll check -- I try to avoid committing this file at all, so I'm not sure what's going on. I mean to only commit changes to pyproject.toml.